src/main/java/nl/tudelft/ti2206/group9/gui/scene/SettingsScene.java

```diff
@@ -23,37 +23,53 @@
     enum BType {
         /** Back button. */
         SETTINGS_BACK,
-        /** Sound toggle. */
-        SETTING_SOUND
+        /** Sound Effects toggle. */
+        SETTING_SOUNDEFFECTS,
+        /** Soundtrack toggle. */
+        SETTING_SOUNDTRACK
      }
 
     /** Toggle width. */
-    private static final int TOGGLE_WIDTH = 150;
+    private static final int TOGGLE_WIDTH = 270;
     /** Toggle text size. */
     private static final int TOGGLE_TEXT_SIZE = 16;
+    /** Initially the toggles are set to off. */
+    private static final String INIT_TOGGLE = "OFF";
```

Comment:
Can you explain why "OFF" is a constant but "ON" is not? Either both should be a constant, or none.

<review,logic